redisinsight/ui/src/components/input-field-sentinel/InputFieldSentinel.tsx

```diff
@@ -76,12 +74,11 @@ const InputFieldSentinel = (props: Props) => {
         />
       )}
       {inputType === SentinelInputFieldType.Number && (
-        <EuiFieldNumber
+        <NumericInput
           {...clearProp}
-          compressed
-          type="text"
-          value={value}
-          onChange={(e) => handleChange(validateNumber(e.target?.value))}
+          autoValidate
+          value={Number(value)}
+          onChange={(value) => handleChange(value ? value.toString() : '')}
```

Comment:
Question: I'm wondering why we store the value as string and when displaying it we convert it to number? I'm not suggesting editing it, just wondered why.

For example, what if:
```suggestion
          value={value}
          onChange={(value) => handleChange(Number(value ? value.toString() : ''))}
```